test/qcron_test.cpp

```diff
@@ -45,6 +48,11 @@ minutes()
     _tnow.setHMS(0, 0, 0);
     QCOMPARE(actual(pattern), now().addSecs(60 * 1));
 
+    // Invalid
```

Comment:
It's been a while since I saw this code, so forgive me if my remark is obviously erroneous, but I'd rather have either all the invalid cases here, or none. I don't see why `/1 * * * * *` would be a special case.

Reply:
I see, updated PR and removed this test.